### ISSUE TITLE
Linter indicating error

### ISSUE THREAD

**Issue by alastairbattrick** [terraform-linters/tflint#330]
Hi
I'm using VSCode and the linter through Version: 1.3.12 of Mikael Olenfalk's Terraform plugin
(https://marketplace.visualstudio.com/items?itemName=mauve.terraform) is indicating an error, despite terraform not having an error when parsing the code.

```
resource "aws_route" "route-vpn-private" {
  count                  = "${length(var.office_subnets) * length(aws_route_table.private)}"
  destination_cidr_block = "${element(element(setproduct(var.office_subnets, aws_route_table.private), count.index),0)}"
  route_table_id         = "${element(element(setproduct(var.office_subnets, aws_route_table.private), count.index),1).id}"
  gateway_id             = "${aws_vpn_gateway.vpn_gateway.id}"
}
```

The route_table_id line is highlighted with 
```
expected "}" but found "."

**Comment by wata727** [comment]
Hi @alastairbattrick

This seems not to be a TFLint issue. I ran TFLint against the file directly and no errors were reported. Please let me know the result of running TFLint directly if you still have an issue. Thanks.

**Comment by wata727** [comment]
Close this due to no activity.
